FrontEnd/package.json

```diff
@@ -9,14 +9,19 @@
     "lint": "eslint src --ext js,jsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview"
   },
+  "license": "MIT",
   "dependencies": {
     "@chakra-ui/react": "^2.6.0",
-    "@emotion/react": "^11.10.8",
-    "@emotion/styled": "^11.10.8",
+    "@emotion/react": "^11.11.0",
+    "@emotion/styled": "^11.11.0",
+    "@mui/icons-material": "^5.11.16",
```

Comment:
Since you have imported `react-icons` and it includes material icons also, why is `@mui/icons-material` required?

Reply:
Removed and Updated 👍